### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jscottcronin**
## Describe the big picture - why are we doing this?
Updating LightFM docs in README to verify the library includes support for side data

## Any additional details to clarify code in the PR?
Do I need to update the Changelog or version files since this update only affects the readme?

## Pull Request Checklist
 - [x] Pull request includes a description of why we are doing this
 - [x] ``CHANGELOG`` has been updated
 - [x] Version in ``_version.py`` has been updated
 - [x] All tests in the ``tests`` folder pass with a local build
 - [x] ``README`` has been updated (if applicable)
 - [x] Documentation in ``docs`` has been updated (if applicable)
 - [x] ``requirements.txt`` and ``requirements-dev.txt`` have been recompiled (if applicable)
 - [x] Docker image can be built using ``docker build -t collie .``

### CODE REVIEW

**Comment by gsganden** [comment]
@nathancooperjones are you able to review?

File: README.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ print(f'MSE: {mse_score}')
 | Includes **model zoo** with two or more model architectures implemented  |  |  |  | ✓ | ✓ |  | ✓ |
 | Includes **implicit loss functions** for training and **metric functions** for model evaluation  |  | ✓ |  | ✓ | ✓ |  | ✓ |
 | Includes **adaptive loss functions** for multiple negative examples  |  | ✓ |  | ✓ |  |  | ✓ |
-| Includes **loss functions that account for side-data**  |  |  |  |  |  |  | ✓ |
+| Includes **loss functions that account for side-data**  |  | ✓ |  |  |  |  | ✓ |
```
Comment (nathancooperjones) [inline README.md:148]: For this point in the `README`, I was specifically thinking about the partial credit loss function stuff that is unique to Collie: 

https://github.com/ShopRunner/collie/blob/fb4bb7a16d61703b2fc9573e70a0bc1f97d27564/collie/loss/metadata_utils.py#L15-L25

My understanding is that this doesn't exist in LightFM - does this seem right to you? 

Maybe we need to update the wording of this row to better reflect what Collie actually brings to the table 🤔

File: README.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ print(f'MSE: {mse_score}')
 | --- | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
 | **Implicit data support** for when we only know when a user interacts with an item or not, not the explicit rating the user gave the item |  | ✓ |  | ✓ | ✓ | ✓ | ✓ |
 | **Explicit data support** for when we know the explicit rating the user gave the item | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
-| Support for **side-data** incorporated directly into the models |  |  |  |  | ✓ | ✓ | ✓ |
+| Support for **side-data** incorporated directly into the models |  | ✓ |  |  | ✓ | ✓ | ✓ |
```
Comment (nathancooperjones) [inline README.md:138]: Great fix!!

**Review by nathancooperjones — COMMENTED** [review]
I left a small question - thanks for fixing this!! 🎉

File: README.md (hunk 1 above)
Comment (jscottcronin) [reply README.md:148]: Yes, let's update the language, because my interpretation was different than what was intended.

File: README.md [hunk 3]
```diff
@@ -145,7 +145,7 @@ print(f'MSE: {mse_score}')
 | Includes **model zoo** with two or more model architectures implemented  |  |  |  | ✓ | ✓ |  | ✓ |
 | Includes **implicit loss functions** for training and **metric functions** for model evaluation  |  | ✓ |  | ✓ | ✓ |  | ✓ |
 | Includes **adaptive loss functions** for multiple negative examples  |  | ✓ |  | ✓ |  |  | ✓ |
-| Includes **loss functions that account for side-data**  |  | ✓ |  |  |  |  | ✓ |
+| Includes **loss functions with [partial credit for side-data](https://github.com/ShopRunner/collie/blob/main/collie/loss/metadata_utils.py#L13-L24)**  |  | ✓ |  |  |  |  | ✓ |
```
Comment (nathancooperjones) [inline README.md:148]: Love the wording here! Now, is it safe to uncheck the box for LightFM?

Comment (nathancooperjones) [reply README.md:148]: Also, can we link to the docs here instead of the code? https://collie.readthedocs.io/en/latest/loss.html

Comment (jscottcronin) [reply README.md]: Updated. Much better link.

**Review by nathancooperjones — APPROVED** [review]
Thank you @jscottcronin!! Now we just need someone from Data Science to also approve and then merge it in.

**Review by mcundiff3 — APPROVED** [review]
Thanks Scott!

**Pull request merged**